interface/patient_tracker/patient_tracker.php

```diff
@@ -168,7 +168,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Remove these two lines of code

Also the flow board should be centered instead of aligned left

Reply:
Also Travis isn't happy lol